be/src/storage/tablet_updates.cpp

```diff
@@ -233,33 +233,67 @@ Status TabletUpdates::_load_from_pb(const TabletUpdatesPB& tablet_updates_pb) {
 
     // Load delete vectors and update RowsetStats.
     // TODO: save num_dels in rowset meta.
+
+    std::map<uint32_t, std::vector<std::pair<int32_t, DelVectorPtr>>> del_vector_by_rsid;
+    uint32_t max_rsid = 0;
+    uint32_t min_rsid = 1 << 31;
+    for (auto& [rsid, rowset] : _rowsets) {
+        if (unapplied_rowsets.find(rsid) == unapplied_rowsets.end()) {
+            std::vector<std::pair<int32_t, DelVectorPtr>> del_vectors;
+            del_vectors.resize(rowset->num_segments());
+            del_vector_by_rsid[rsid] = del_vectors;
```

Comment:
del_vector_by_rsid[rsid].resize(rowset->num_segments())
and maybe you should make sure version is initialized as 0?

Reply:
> del_vector_by_rsid[rsid].resize(rowset->num_segments()) and maybe you should make sure version is initialized as 0?

Good advice, changed in https://github.com/StarRocks/starrocks/pull/19347/commits/7bf7ae3af2f36089321a824401b85d2f42945935